Makefile

```diff
@@ -167,9 +190,21 @@ release-upload-cluster-controller: release-cluster-controller upload-artifacts
 upload-artifacts:
 	controllers/build/upload_artifacts.sh $(TAR_PATH) $(ARTIFACTS_BUCKET) $(PROJECT_PATH) $(CODEBUILD_BUILD_NUMBER) $(CODEBUILD_RESOLVED_SOURCE_VERSION)
 
-.PHONY: cluster-controller-binaries
-cluster-controller-binaries:
-	controllers/build/create_binaries.sh $(GOLANG_VERSION) $(BINARY_NAME) $(IMAGE_REPO) $(IMAGE_TAG)
+PLATFORMS = linux-amd64 linux-arm64
+create-cluster-controller-binaries: $(foreach platform,$(PLATFORMS),create-cluster-controller-binary-$(platform))
+
+create-cluster-controller-binary-%:
+	CGO_ENABLED=0 GOOS=$(firstword $(subst -, ,$*)) GOARCH=$(lastword $(subst -, ,$*)) $(MAKE) build-cluster-controller-binaries
```

Comment:
if you are game for it, in another PR, would love to see this for the eks-a binaries builds themselves as well

Reply:
I went ahead and pushed this change in the PR as well